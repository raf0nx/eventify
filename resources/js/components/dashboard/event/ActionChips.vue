<template>
	<div class="my-8">
		<v-chip
			class="mr-2 font-weight-bold"
			color="deep-purple"
			dark
			@click.stop="$emit('openDialog')"
		>
			<v-icon left> mdi-plus-circle </v-icon>
			Create
		</v-chip>
		<v-chip class="mr-2 font-weight-bold" color="deep-purple" outlined>
			<v-icon left> mdi-calendar </v-icon>
			Your events
		</v-chip>
		<v-chip class="mr-2 font-weight-bold" color="deep-purple" dark>
			<v-icon small> mdi-bell </v-icon>
		</v-chip>
	</div>
</template>

<script lang="ts">
	import { Vue, Component } from "vue-property-decorator";

	@Component
	export default class ActionChips extends Vue {}
</script>

<style>
</style>