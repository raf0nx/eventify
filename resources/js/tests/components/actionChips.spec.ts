import { createLocalVue, shallowMount, Wrapper } from "@vue/test-utils";

import ActionChips from "@/components/dashboard/event/ActionChips.vue";

describe("ActionChips.vue", () => {
    const localVue = createLocalVue();
    let wrapper: Wrapper<ActionChips>;

    beforeEach(() => {
        wrapper = shallowMount(ActionChips, { localVue });
    });

    afterEach(() => {
        wrapper.destroy();
    });

    it("Should match snapshot", () => {
        // Assert
        expect(wrapper.html()).toMatchSnapshot();
    });
});
