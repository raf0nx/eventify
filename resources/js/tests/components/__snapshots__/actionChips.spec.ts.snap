// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionChips.vue Should match snapshot 1`] = `
"<div class=\\"my-8\\">
  <v-chip-stub color=\\"deep-purple\\" activeclass=\\"\\" tag=\\"span\\" dark=\\"true\\" active=\\"true\\" closeicon=\\"$delete\\" closelabel=\\"$vuetify.close\\" filtericon=\\"$complete\\" class=\\"mr-2 font-weight-bold\\">
    <v-icon-stub left=\\"\\"> mdi-plus-circle </v-icon-stub>
    Create
  </v-chip-stub>
  <v-chip-stub color=\\"deep-purple\\" activeclass=\\"\\" tag=\\"span\\" active=\\"true\\" closeicon=\\"$delete\\" closelabel=\\"$vuetify.close\\" filtericon=\\"$complete\\" outlined=\\"true\\" class=\\"mr-2 font-weight-bold\\">
    <v-icon-stub left=\\"\\"> mdi-calendar </v-icon-stub>
    Your events
  </v-chip-stub>
  <v-chip-stub color=\\"deep-purple\\" activeclass=\\"\\" tag=\\"span\\" dark=\\"true\\" active=\\"true\\" closeicon=\\"$delete\\" closelabel=\\"$vuetify.close\\" filtericon=\\"$complete\\" class=\\"mr-2 font-weight-bold\\">
    <v-icon-stub small=\\"\\"> mdi-bell </v-icon-stub>
  </v-chip-stub>
</div>"
`;
